### PR TITLE
Add Finn's member page

### PR DESCRIPTION
**Pull request by premfi**
This website is based on the Lab Website Template.
See its documentation for working with this site:

https://greene-lab.gitbook.io/lab-website-template-docs

### CODE REVIEW

File: _data/members-list.yaml [hunk 1]
```diff
@@ -83,6 +83,12 @@ Roman Remme:
   phone: +49 6221 5414802
   address: Im Neuenheimer Feld 205, 69120 Heidelberg
 
+Finn Prem:
+  email: finn.prem@stud.uni-heidelberg.de
+  room: Room 4.305
+  phone: +49 6221 0000000
```
Comment (christofgehrig) [inline _data/members-list.yaml:89]: You should add the phone number of the telephone in your office.

File: team/index.md [hunk 2]
```diff
@@ -23,13 +23,6 @@ We are physicists, mathematicians, computer scientists and engineers, united by
 {% include list.html data="members" component="portrait" filters="role: visiting-prof" %}
 {% include list.html data="members" component="portrait" filters="role: ^(?!pi$|visiting-prof$)" %}
 
-### Members who don't have a member page yet
-{:.center}
-
-| Name  | Role  |
-| :---- | :---- |
-| Finn Prem | BSc Student |
-
```
Comment (christofgehrig) [inline team/index.md:32]: We should comment this out so that we can add it again easily. Or actually, we could note down Lennart Bürger (MSc Student).

**Review by christofgehrig — CHANGES_REQUESTED** [review]

File: team/index.md (hunk 2 above)
Comment (premfi) [reply team/index.md]: Ok, I added him.

File: _data/members-list.yaml (hunk 1 above)
Comment (premfi) [reply _data/members-list.yaml]: Ok, I added it now.

**Review by christofgehrig — APPROVED** [review]

**Pull request merged**